Index: css/common.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- css/common.css	(revision d9b5133196c6c0f30cfa6b8a9f0ad503fb0ade4c)
+++ css/common.css	(revision )
@@ -47,15 +47,25 @@
 
 /*ClassifyBar*/
 .classifyIcon{
-    /*padding: 30px;*/
-}
-.classifyIcon img{
     width: 120px;
     height: 120px;
+    border-radius: 30px;
+    background-color: #e6e6e6;
+    border: 1px solid black;
+    position: relative;
 }
-.classifyIcon p{
-    font-size: 30px;
+.classifyIcon img{
+    width: 80px;
+    height: 80px;
+    position: absolute;
+    left: 20px;
+    top: 20px;
+}
+.classifyIconP{
+    font-size: 22px;
     text-align: center;
+    width: 120px;
+    padding-top: 15px;
 }
 .classifyBox{
     display: flex;
Index: src/component/main.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/component/main.js	(revision d9b5133196c6c0f30cfa6b8a9f0ad503fb0ade4c)
+++ src/component/main.js	(revision )
@@ -22,31 +22,31 @@
 import OrderConfirm from './Orders/Confirm/OrderConfirm'
 import Intro from './Information/Intro'
 
-import {Switch, BrowserRouter as Router, Route, Redirect,Link  } from 'react-router-dom';
-import createHistory from 'history/createBrowserHistory'
+import {Switch, HashRouter as Router, Route, Redirect, Link} from 'react-router-dom';
+import createHistory from 'history/createHashHistory'
 import {Provider} from 'react-redux';
 import configureStore from '../store/configureStore';
 import injectTapEventPlugin from 'react-tap-event-plugin';
 injectTapEventPlugin();
 
 
-
 let store = configureStore();
 const history = createHistory();
 
-class Main extends React.Component{
-    constructor(props){
+class Main extends React.Component {
+    constructor(props) {
         super(props)
     }
 
-    render(){
-            return(
-                <Provider store={store}>
-                    <Router history={history}>
-                        <div>
+    render() {
+        return (
+            <Provider store={store}>
+                <Router history={history}>
+                    <div>
+                        <Switch>
                             <Route path="/information/:shopId/:id" component={Intro}/>
                             <Route path="/home/:id" component={BaseOptions}/>
-                            <Route  path="/shop/:id" component={ShopPage}/>
+                            <Route path="/shop/:id" component={ShopPage}/>
                             <Route path="/search/:id" component={SearchIndex}/>
                             <Route path="/address/:id" component={AddressMain}/>
                             <Route path="/delivery" component={Delivery}/>
@@ -59,15 +59,16 @@
                             <Route path="/payment/succeed" component={PaymentSuccess}/>
                             <Route path="/payment/failed" component={PaymentFail}/>
                             <Route path="/orderConfirm" component={OrderConfirm}/>
-
+                            <Redirect exact from="/" to="/home/index"/>
+                        </Switch>
 
-                        </div>
-                    </Router>
-                </Provider>
-            )
+                    </div>
+                </Router>
+            </Provider>
+        )
 
     }
 }
 
-ReactDOM.render(<Main/>,document.getElementById("app"));
+ReactDOM.render(<Main/>, document.getElementById("app"));
 
Index: src/component/Home/ClassifyBar/ClassifyBarItem.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/component/Home/ClassifyBar/ClassifyBarItem.js	(revision d9b5133196c6c0f30cfa6b8a9f0ad503fb0ade4c)
+++ src/component/Home/ClassifyBar/ClassifyBarItem.js	(revision )
@@ -14,10 +14,12 @@
 
     render() {
         return (
-            <span className="classifyIcon">
-                <img src={this.props.src.src}/>
-                <p>{this.props.src.detail}</p>
-            </span>
+           <div>
+               <div className="classifyIcon">
+                   <img src={this.props.src.src}/>
+               </div>
+               <p className="classifyIconP">{this.props.src.detail}</p>
+           </div>
         )
     }
 }
\ No newline at end of file
